### PR TITLE
Workflow to Automatically Title Case PRs

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

Hooray, more JavaScript!
I tested this on a personal repo with a new PAT, but it *should* work here first try.

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]
Very Nice, Much Title Case, Wow.

File: Tools/changelog/package.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
     "name": "changelogs",
+    "author": "DEATHB4DEFEAT",
+    "license": "MIT",
```
Comment (DangerRevolution) [inline Tools/changelog/package.json:4]: Is this okay to have in an AGPL codebase?

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
Just one question

File: Tools/changelog/package.json (hunk 1 above)
Comment (DEATHB4DEFEAT) [reply Tools/changelog/package.json:4]: We have WizDen in here, of course.

Comment (DangerRevolution) [reply Tools/changelog/package.json]: Yeah, but that naturally got relicensed no? Idrk how this business works

**Review by DangerRevolution — APPROVED** [review]
Idrk how licensing works; looks fine to me

**Comment by DangerRevolution** [comment]
Will merge when tests pass

File: Tools/changelog/package.json (hunk 1 above)
Comment (stellar-novas) [reply Tools/changelog/package.json]: Same thing here, you can include MIT code in AGPL codebases

**Pull request merged**